sklearn/decomposition/_truncated_svd.py

```diff
@@ -113,7 +113,8 @@ class TruncatedSVD(TransformerMixin, BaseEstimator):
     sign of the ``components_`` and the output from transform depend on the
     algorithm and random state. To work around this, fit instances of this
     class to data once, then keep the instance around to do transformations.
-
+    
+    ..versionadded:: v0.14
```

Comment:
I think we usually don't add the `v` (here and there is your PR)

Reply:
This branch is a compilation of several different groups, so it looks like some of them put in v for version, but I agree it shouldn't be there. Should I go through and fix the formatting or is there a better process for handling that?

Reply:
Yes you should fix them